tests/helpers.py

```diff
@@ -12,12 +12,53 @@
 import copy
 import uuid
 
+from flask import current_app
 from invenio_db import db
 from invenio_pidstore import current_pidstore
 
 from invenio_circulation.api import Loan
 
 
+class SwappedConfig:
```

Comment:
scope function wasn't enough? Just out of curiosity?

Reply:
I did that just because I find nice to have the with statement, to have a temporary context that puts back changes when exiting. Feel free to change to what u prefer!
Also, maybe better to merge 2 classes in one, but I didn't want to have unreadable code...